appyx-components/stable/spotlight/common/src/commonMain/kotlin/com/bumble/appyx/components/spotlight/ui/slider/SpotlightSlider.kt

```diff
@@ -29,12 +29,16 @@ import com.bumble.appyx.transitionmodel.BaseVisualisation
 
 class SpotlightSlider<InteractionTarget : Any>(
     uiContext: UiContext,
+    initialState: State<InteractionTarget>,
```

Comment:
Just to clarify, here the naming is correct, since the slider will be initialised based on this state. Naming it here as `currentState` would be also misleading since it does not keep track of changes.

I assume this naming was the source of confusion outside of this class.